_source/user-guide/infrastructure-monitoring/custom-metrics.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: article
+title: Custom metrics
+permalink: /user-guide/infrastructure-monitoring/custom-metrics
+flags:
+  admin: true
+  #logzio-plan: pro
+tags:
+  - Grafana
+contributors:
+  - shalper
+---
+
+You can send your application metrics to your Logz.io Infrastructure Monitoring account and benefit from all of the advantages of your tried and true dashboards.
+
+To do this, you'll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
+
+##### Overview
+{:.no_toc}
+
+1. toc list
+{:toc}
+
+#### JSON format
+{:.no_toc}
+
+Our guiding principle is to use key-value pairs to define metrics and their dimensions.
+Here's the general template.
+
+  ```
+  {
+	"metrics": {
+		"my_first_metric.unit": number,
+		"my_second_metric.unit": number,
+		"my_third_metric.unit": number,
+		"my_fourtn_metrlc.unit": number
+	},
+	"dimensions": {
+		"dimension_l": "value",
+		"dimension_2": "value",
+		"dimension_3": "value",
+		"dimension_4": "value"
+	}
+  }
+  ```
+
+The reason being that we want to send _fewer and longer_ documents. 
+By grouping all possible metrics and dimensions into one document we maximize Elasticsearch's indexing power. In general, the _number_ of documents has a greater impact on Elasticsearch indexing power than document length.
+
+The best way to explain the guidelines is to look at a few examples. So let's get to it.
```

Comment:
I think you should stress that these are a few example of possible client's custom metrics arrangement *before* Logz.io is even in the picture. Some people may think these are example of how we guide them to arrange their data.

Reply:
Not sure what you mean. Please suggest an edit or remove the parts you don't like.